src/main/java/com/vdovich/tbfm/nasaAPI/NasaController.java

```diff
@@ -0,0 +1,48 @@
+package com.vdovich.tbfm.nasaAPI;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.URL;
+import java.net.URLConnection;
+
+import org.json.JSONException;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/nasa")
+public class NasaController {
+    JsonParser jsonParser = new JsonParser();
+
+    @GetMapping("/pictureOfTheDay")
+    public String callNasaApi() throws IOException, JSONException {
```

Comment:
Do not throw the exceptions. You need to handle it with appropriate logging and return error message.